source/Applications/Advanced/MaskPointCloud/MaskPointCloud.cpp

```diff
@@ -0,0 +1,224 @@
+                    maskedPointCloud(j, i).g = data(i, j).color.g;
+                    maskedPointCloud(j, i).b = data(i, j).color.b;
+                    maskedPointCloud(j, i).x = data(i, j).point.x;
+                    maskedPointCloud(j, i).y = data(i, j).point.y;
+                    maskedPointCloud(j, i).z = data(i, j).point.z;
+                }
+                else
+                {
+                    maskedPointCloud(j, i).r = 0;
+                    maskedPointCloud(j, i).g = 0;
+                    maskedPointCloud(j, i).b = 0;
+                    maskedPointCloud(j, i).x = NAN;
+                    maskedPointCloud(j, i).y = NAN;
+                    maskedPointCloud(j, i).z = NAN;
+                }
+            }
+        }
+
+        return maskedPointCloud;
+    }
+
+    void visualizeDepthMap(const pcl::PointCloud<pcl::PointXYZRGB> &pointCloud)
+    {
+        std::cout << "Converting to Depth map in OpenCV format" << std::endl;
+        cv::Mat zJetColorMap = pointCloudToCvZ(pointCloud);
+        std::cout << "Visualizing Depth map" << std::endl;
+        cv::namedWindow("Depth map", cv::WINDOW_AUTOSIZE);
+        cv::imshow("Depth map", zJetColorMap);
+        cv::waitKey(0);
+    }
+
+    pcl::PointCloud<pcl::PointXYZRGB> convertToPCLPointCloud(const Zivid::PointCloud &pointCloud)
+    {
+        const auto data = pointCloud.copyData<Zivid::PointXYZColorRGBA>();
+
+        std::cout << "Creating PCL point cloud structure" << std::endl;
+        pcl::PointCloud<pcl::PointXYZRGB> pointCloudPCL;
+
+        std::cout << "Filling in point cloud data" << std::endl;
+        pointCloudPCL.width = pointCloud.width();
+        pointCloudPCL.height = pointCloud.height();
+        pointCloudPCL.is_dense = false;
+        pointCloudPCL.points.resize(pointCloudPCL.width * pointCloudPCL.height);
+
+        for(size_t i = 0; i < pointCloudPCL.points.size(); ++i)
+        {
+            pointCloudPCL.points[i].x = data(i).point.x; // NOLINT(cppcoreguidelines-pro-type-union-access)
+            pointCloudPCL.points[i].y = data(i).point.y; // NOLINT(cppcoreguidelines-pro-type-union-access)
+            pointCloudPCL.points[i].z = data(i).point.z; // NOLINT(cppcoreguidelines-pro-type-union-access)
+            pointCloudPCL.points[i].r = data(i).color.r; // NOLINT(cppcoreguidelines-pro-type-union-access)
+            pointCloudPCL.points[i].g = data(i).color.g; // NOLINT(cppcoreguidelines-pro-type-union-access)
+            pointCloudPCL.points[i].b = data(i).color.b; // NOLINT(cppcoreguidelines-pro-type-union-access)
+        }
+        return pointCloudPCL;
+    }
+
+} // namespace
+
+int main()
+{
+    try
+    {
+        Zivid::Application zivid;
+
+        std::string fileName = std::string(ZIVID_SAMPLE_DATA_DIR) + "/Zivid3D.zdf";
+        std::cout << "Reading ZDF frame from file: " << fileName << std::endl;
+        const auto frame = Zivid::Frame(fileName);
+        const auto pointCloud = frame.pointCloud();
```

Comment:
Add an empty line as we have in other samples, see: https://github.com/zivid/zivid-cpp-samples/blob/mask_point_cloud/source/Applications/Basic/FileFormats/ReadIterateZDF/ReadIterateZDF.cpp

Also, print out `std::cout << "Getting point cloud from frame" << std::endl;` before this line.